x-pack/plugin/src/test/resources/rest-api-spec/api/async_search.submit.json

```diff
@@ -0,0 +1,226 @@
+{
+  "async_search.submit":{
+    "documentation":{
+      "url":"https://www.elastic.co/guide/en/elasticsearch/reference/current/async-search.html"
+    },
+    "stability":"experimental",
+    "url":{
+      "paths":[
+        {
+          "path":"/_async_search",
+          "methods":[
+            "GET",
+            "POST"
+          ]
+        },
+        {
+          "path":"/{index}/_async_search",
+          "methods":[
+            "GET",
+            "POST"
+          ],
+          "parts":{
+            "index":{
+              "type":"list",
+              "description":"A comma-separated list of index names to search; use `_all` or empty string to perform the operation on all indices"
+            }
+          }
+        }
+      ]
+    },
+    "params":{
+      "wait_for_completion":{
+        "type":"time",
+        "description":"Specify the time that the request should block waiting for the final response (default: 1s)"
+      },
+      "clean_on_completion":{
+        "type":"boolean",
+        "description":"Specify whether the response should not be stored in the cluster if it completed within the provided wait_for_completion time (default: true)"
+      },
+      "batched_reduce_size":{
+        "type":"number",
+        "description":"The number of shard results that should be reduced at once on the coordinating node. This value should be used as the granularity at which progress results will be made available.",
+        "default":5
+      },
+      "keep_alive": {
+        "type": "time",
+        "description": "Specify the time that the request should remain reachable in the cluster."
```

Comment:
maybe rephrase to something like "Specify the time interval in which the results (partial or final) for this search will be available"